axonius_api_client/cli/grp_assets/grp_common.py

```diff
@@ -169,6 +170,32 @@ def wiz_callback(ctx, param, value):
     return "\n".join(contents)
 
 
+def expirable_tags_callback(ctx, param, values) -> dict:
+    expirable_tags: dict = {}
+
+    if not values:
+        return expirable_tags
+
+    for value in values:
+        ctx.obj.echo_ok(f'value {value}')
+        if "=" not in value:
+            echo_error(msg=f"Invalid expression, must contain tag_name=expired_date or tag_name=days_from_now")
+
+        tag_name, tag_expiration_date = value.split('=')
+        if isinstance(tag_expiration_date, int) or tag_expiration_date.isdigit():
+            tag_expiration_date = int(tag_expiration_date)
+            tag_expiration_date = str(datetime.utcnow().date() + timedelta(days=tag_expiration_date))
```

Comment:
This does not seem correct.

If I put in `2024-03-03` as a date for expiration within 5 hours of midnight, it will end up on the server as `2024-03-02` because i'm GMT-5. I do not think we can get away with just using a date without the time.

I think we need to figure out the timezone offset hours of the local time and add them back to the UTC time.
This appears to be what the GUI does. If I add a tag, it does a datetime utc date with my GMT offset added to the hours. `{name: "TestTag", expiration_date: "2024-02-06T05:00:00.000Z"}`.

Reply:
@OzJacobii  ^

Reply:
Hey @nate-axonius 
I converted it to be the date of the user, that way it will save the date as the date of the user and no need to consider UTC calculations
- add the offset hours of local time to the UTC == local datetime of the user